src/layouts/Home/components/Search.tsx

```diff
@@ -118,12 +117,8 @@ export const Search: React.FC = () => {
                     }
                   `}
                 >
-                  <MediaInput
-                    placeholder="What is kale-600?"
-                    aria-label="Garden search"
-                    start={<SearchStroke />}
-                  />
-                </div> */}
+                  <SearchInput id="algolia-docsearch-home" placeholder="kale-600 hex value" />
```

Comment:
since we don't want to encourage people to use the hex vales maybe we change this to: "how to use the stepper"

As an aside, it could be interesting to have this rotate between a bunch of possible placeholders; could also be a subtle way to advertise new components or patterns.

Reply:
I was thinking the same thing 👍 Thanks, @allisonacs